### PR TITLE
ROC-3794 Upgrade pa11y to latest version (5.0.4)

### PR DESCRIPTION
**Pull request by DamianDunajski**
### JIRA link

None

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/test/a11y/a11y.ts [hunk 1]
```diff
@@ -23,23 +23,31 @@ app.locals.csrf = 'dummy-token'
...
+  type
+}
+
+async function runPa11y (url: string): Promise<Issue[]> {
+  const result = await pa11y(url, {
     headers: {
       Cookie: `${cookieName}=ABC`
+    },
+    chromeLaunchConfig: {
+      args: ['--no-sandbox']
```
Comment (geek-soft) [inline src/test/a11y/a11y.ts]: Needed to run puppeteer on Travis without any additional changes to the build process.

See: https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-on-travis-ci

**Comment by robertparkinson** [comment]
there is a jira ticket for this: 
https://tools.hmcts.net/jira/browse/ROC-3794

File: package.json [hunk 2]
```diff
@@ -126,7 +125,7 @@
     "nock": "^9.0.22",
     "nsp": "^3.0.0",
     "nyc": "^12.0.2",
-    "pa11y": "^4.6.0",
+    "pa11y": "^5.0.0",
```
Comment (robertparkinson) [inline package.json]: shouldn't this be 5.0.4?

**Comment by robertparkinson** [comment]
extra challenge.. can you make them work in threads like the other tests? ;)

File: package.json (hunk 2 above)
Comment (geek-soft) [reply package.json]: Could be, but it does not really matter because yarn.lock pins to 5.0.4 (latest version in range >=5.0.0 and <6.0.0.

**Pull request merged**